pkg/controller/gitjob/gitjobs.go

```diff
@@ -32,6 +32,7 @@ const (
 	bundleCAFile            = "additional-ca.crt"
 	gitCredentialVolumeName = "git-credential" // #nosec G101 this is not a credential
 	gitClonerVolumeName     = "git-cloner"
+	emptyDirVolumeName      = "empty-dir"
```

Comment:
Can we use a more specific name, like `git-cloner-empty-dir` or `git-cloner-tmp-dir`?

Reply:
changed in https://github.com/rancher/gitjob/pull/331/commits/2fab34e0cfbc0d37e9b38233456eb9253922647f